src/common/mem.c

```diff
@@ -2237,11 +2237,13 @@ cache_pop_free_page(struct umem_cache *cache)
 #define UMEM_CACHE_BMAP_SZ_MAX    (1 << (UMEM_CACHE_PAGE_SHIFT_MAX - \
 					UMEM_CACHE_CHUNK_SZ_SHIFT - UMEM_CHUNK_IDX_SHIFT))
 #define UMEM_CACHE_RSRVD_PAGES	4
+#define UMEM_CACHE_MIN_EVICTABLE_PAGES	2
```

Comment:
Can this be moved to mem.h so that this macro can be accessed from allocator

Reply:
sure.